static/app/components/events/userFeedback.tsx

```diff
@@ -11,6 +11,36 @@ import type {UserReport} from 'sentry/types/group';
 import {escape, nl2br} from 'sentry/utils';
 import useCopyToClipboard from 'sentry/utils/useCopyToClipboard';
 
+// used for issue details page (groupEventDetailsContent.tsx)
+export function HideUserFeedbackButton({
```

Comment:
I'm thinking of moving this to groupEventDetailsContent.tsx

Reply:
might be good, could reduce a bunch of prop passing